docs/reference/query-languages/esql/kibana/definition/settings/project_routing.json

```diff
@@ -0,0 +1,7 @@
+{
+  "comment" : "This is generated by ESQL’s DocsV3Support. Do not edit it. See ../README.md for how to regenerate it.",
+  "name" : "project_routing",
+  "type" : "keyword",
```

Comment:
What is this about Luigi?

Reply:
It's the allowed parameters for SET statement: 

```
SET project_routing="<keyword-routing-expression>"; FROM idx | ...
```

For now we have only `project_routing`, but maybe we'll have more in the future.

If you think any additional info could be useful for client-side validation please let me know

Reply:
It makes sense IMHO, I'm adding them